src/components/shared/Reorderable.js

```diff
@@ -19,7 +19,17 @@ type Props = {|
   className: string,
   order: number[],
   onChangeOrder: (number[]) => Action,
-  children?: React$Element<*>,
+  children: React.ChildrenArray<React.Element<any>>,
```

Comment:
This typing forces children to be an array of react elements.
See https://flow.org/en/docs/react/children/ for a lengthy documentation about Childrens. Especially https://flow.org/en/docs/react/children/#toc-using-react-node-but-without-some-primitive-types-like-strings